src/AFQMC/Estimators/BackPropagatedEstimator.hpp

```diff
@@ -191,7 +191,7 @@ class BackPropagatedEstimator : public EstimatorBase
 
     // 1. check structures
     if (Refs.size(0) != wset.size() || Refs.size(1) != nrefs || Refs.size(2) != nrow * ncol)
-      Refs = std::move(mpi3CTensor({wset.size(), nrefs, nrow * ncol}, Refs.get_allocator()));
```

Comment:
In this case, temporary mpi3CTensor type object is created and moved to Refs via move assign operator.